environments/development.env

```
@@ -4,7 +4,7 @@ BASE_URL=http://localhost:8000
 WEBPACK_DEV_URL=http://localhost:8001
 NODE_ENV=development
 LP_API_URL=https://api.staging.launchpad.net
-WEBHOOK_ENDPOINT=https://build.staging.snapcraft.io/webhook/notify
+WEBHOOK_ENDPOINT=http://localhost:8000/api/webhook/notify
```

Default to the mock service.